pusher/src/Services/OpenIDClient.ts

```diff
@@ -25,8 +28,11 @@ class OpenIDClient {
 
     public authorizationUrl(state: string, nonce: string, playUri?: string, redirect?: string) {
         return this.initClient().then((client) => {
+            if( OPID_SCOPE == undefined || OPID_SCOPE.indexOf('email') === -1 || OPID_SCOPE.indexOf('openid') === -1 ) {
```

Comment:
if you define:
```
export const OPID_SCOPE = process.env.OPID_SCOPE || "openid email";
```

`OPID_SCOPE` cannot to be equal to `undefined `

Reply:
why then `if( OPID_SCOPES != undefined && (` before? Due to code checking?

Reply:
I think you can delete `OPID_SCOPES != undefined` it's not necessary with:
```js
export const OPID_SCOPE = process.env.OPID_SCOPE || "openid email";
```